packages/wonder-blocks-button/src/components/button-core.tsx

```diff
@@ -416,6 +420,25 @@ export const _generateStyles = (
                 ? theme.color.bg.secondary.active.critical
                 : theme.color.bg.secondary.active.action;
 
+        const focusStyling = {
+            background: light
+                ? theme.color.bg.secondary.inverse
+                : theme.color.bg.secondary.focus,
+            borderColor: "transparent",
+            outlineColor: light ? theme.color.border.primary.inverse : color,
+            outlineStyle: "solid",
+            outlineWidth: theme.border.width.focused,
+        };
+
+        const activePressedStyling = {
+            background: light ? activeColor : secondaryActiveColor,
+            color: light ? fadedColor : activeColor,
+            borderColor: "transparent",
+            outlineColor: light ? fadedColor : activeColor,
+            outlineStyle: "solid",
+            outlineWidth: theme.border.width.focused,
+        };
```

Comment:
It'll be so nice when we can share state styling across different components 😄

Reply:
Yess!!! 1000% to that 🙏